docs/core/index.md

```diff
@@ -119,15 +119,15 @@ Import existing projects into your nx workspace
 
 ### [add-swagger-target](./generators/add-swagger-target.md)
 
-Generate a target to extract the swagger.json file from a .NET webapi
+Add a swagger target to a webapi based project to extract swagger.json into a newly generated library project
 
 ### [swagger-typescript](./generators/swagger-typescript.md)
 
-Generates typescript code based on a specified openapi/swagger json file
+Generate a typescript library project based on an openapi/swagger specification file
```

Comment:
This generator only performs the codegen itself, it doesn't create a new library. Honestly, it should probably have `hidden: true`, and nx-doc should be updated to hide those. But that's not supported currently.

For now, I'd just make sure this description doesn't sound like it generates a project.